.travis.yml

```diff
@@ -0,0 +1,75 @@
+language: php
+
+php:
+  - '7.1'
+  - '7.2'
+  - '7.3'
+
+sudo: required
+
+services:
+  - docker
+  - mysql
+
+mysql:
+  database: sparql_entity_storage_test
+  username: root
+  encoding: utf8
+
+env:
+  global:
+    - SITE_DIR="$HOME/build/testing_site"
+  matrix:
+    - TEST=PHP_CodeSniffer
+    - TEST=8.6.x
+    - TEST=8.7.x
+    - TEST=8.8.x
```

Comment:
These versions of Drupal are not supported anymore. Instead, we need 8.9.x, 9.0.x, 9.1.x, 9.2.x. Why not replicating https://github.com/ec-europa/rdf_entity/blob/8.x-1.x/.travis.yml which was update recently?